tests/Fake/FakeDomain.php

```diff
@@ -11,7 +11,7 @@ class FakeDomain implements DomainInterface
     public function __invoke(array $input)
     {
         return (new Payload())
-            ->withStatus(Payload::OK)
```

Comment:
May be a good idea to note the PR that made this change for posterity, as it doesn't appear to be mentioned anywhere in this one presently.